masterlist.yaml

```
@@ -12894,20 +12894,31 @@ plugins:
         util: '[TES4Edit v4.0.3h (Hotfix 1)](https://www.nexusmods.com/oblivion/mods/11536)'
         udr: 3
 
+  - name: 'GoldenCrest(BattlehornComp)?\.esp'
+    url:
+      - link: 'https://www.nexusmods.com/oblivion/mods/33626/'
+        name: 'The Golden Crest - A Major Pirate Quest'
   - name: 'GoldenCrest.esp'
-    url: [ 'https://www.nexusmods.com/oblivion/mods/33626/' ]
+    req: [ 'DLCShiveringIsles.esp' ]
```

> Requirements:
> - OBSE -Elys- Universal Silent Voice which can be found here: http://www.tesnexus.com/downloads/file.php?id=16622
(Alternatively you can download the Golden Crest silent mp3 from this site, included in the file section of the mod and extract the files to your OblivionDataSoundVoice directory).

You could add a silent voice plugin requirement as well & hide it when the optional voice files are installed.
```suggestion
    req:
      - <<: *SilentVoice
        condition: 'not file("OBSE/Plugins/voicefile_redirector.dll") and not file("Sound/Voice/GoldenCrest.esp")'
      - 'DLCShiveringIsles.esp'
```